RegistrationController.java

```diff
@@ -0,0 +1,56 @@
+package com.application.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.application.model.Teacher;
+import com.application.model.User;
+import com.application.services.TeacherService;
+import com.application.services.UserService;
+
+@RestController
+public class RegistrationController 
+{
+	@Autowired
+	private UserService userService;
+	
+	@Autowired
+	private TeacherService teacherService;
+	
+	//register user

```

Comment:
Please add mapping file for register user.

Reply:
I added the Post Mapping for register user and upload the source file into my Repository. Could you recheck my source file

Reply:
Ok you can update.

Reply:
Ok sure